poetry.lock

```diff
@@ -0,0 +1,1906 @@
+[[package]]
```

Comment:
@zakaprov unless we have a good reason, I would advise against committing a lockfile to the repo... unless our aim is to completely freeze the dependencies (at the cost of additional maintenance needed when we do want to update them) - to, e.g. protect against sudden, breaking updates of one of the dependencies, it's better for poetry to be able to resolve them from the requirements in `pyproject.toml`